language/resource.language.en_gb/strings.po

```diff
@@ -1459,3 +1459,18 @@ msgstr ""
 msgctxt "#31289"
 msgid "Use local library info for recommendations"
 msgstr ""
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31290"
+msgid "Stay on home menu"
```

Comment:
Not a big deal but was thinking if this label is a bit more generic like `Do nothing` then it can be reused.

Reply:
Changed to `Do nothing`